### ISSUE TITLE
Few first messages do not arrive when there is more then 300 connections

### ISSUE THREAD

**Issue by lwilczak** [dimikot/dklab_realplexor#32]
Hi, i'm working on chat, which is based on realplexor. When there is about 200 users (connections) online everything is ok, but when its growing and its 300+ then for first few seconds messages do not arrive for new logged in user.
Generally: when there is 300+ connections to server, new connection is created, then for few seconds messages sent to new connection do not arrive. 
Administrator of server says, that process of realplexor uses 100% processor.

What can i do with it, how can i debug it? Do you think, upgrade to newest version could help?

**Comment by dimikot** [comment]
Do you use С++ version?
(Perl version is only a prototype, not for production.)

**Comment by lwilczak** [comment]
Yes, its Perl version. It was applied by previous developer over 2 years ago. Thanks for suggestion. Will change version and will see it helped. Thanks

**Comment by lwilczak** [comment]
We changed version to C++, and everything now works fine! Proces use processor normally, nad there is no missing messages. Thanks for help.

**Comment by dimikot** [comment]
I also recommend to setup nginx in front of realplexor, because realplexor
does not support tcp output stream buffering for now (it's in todo).
Such buffering
is a subject nginx is good at.
